app/src/main/java/com/example/cocktails/ui/main/SectionsPagerAdapter.kt

```diff
@@ -21,7 +21,7 @@ class SectionsPagerAdapter(private val context: Context, fm: FragmentManager) :
     override fun getItem(position: Int): Fragment {
         // getItem is called to instantiate the fragment for the given page.
         // Return a PlaceholderFragment (defined as a static inner class below).
-        if (position == 0) {
+        if (position == 1) {
```

Comment:
@shahafhermann cool, but you need to change the permission request location - now i'm being asked for the AR permission in the first tab (the recipe tab)

Reply:
That's very strange since the permission request only happens when initializing the ARFragment. Switching the tab location was merely just changing the order of the titles in the global field, so it shouldn't have anything to do with the permission request. I'll look into it (opened an issue).